docs/protocol-design/ledger.md

```diff
@@ -6,16 +6,72 @@ Title: Nano Protocol Design - Ledger
+
+``` mermaid
+graph TB
+    subgraph Account C
+    A["Block <i>N<sub>C</sub></i>"]-->B["Block <i>N<sub>C</sub></i>  - 1"]
+    B-->C["..."]
+    C-->D["Block 1"]
+    D-->E["Block 0"]
+    end    
+    subgraph Account B
+    F["Block <i>N<sub>B</sub></i>"]-->G["Block <i>N<sub>B</sub></i>  - 1"]
+    G-->H["..."]
+    H-->I["Block 1"]
+    I-->J["Block 0"]
+    end
+    subgraph Account A
+    K["Block <i>N<sub>A</sub></i>"]-->L["Block <i>N<sub>A</sub></i> - 1"]
+    L-->M["..."]
+    M-->N["Block 1"]
+    N-->O["Block 0"]
+    end
+```
+*(Figure 1.  Each account has its own blockchain containing the account’s balance history. Block 0 must be an open transaction)*
+
 ### Accounts
 
+An account is the public-key portion of a digital signature key-pair. The public-key, also referred to as the address, is shared with other network participants while the private-key is kept secret. A digitally signed packet of data ensures that the contents were approved by the private-key holder. One user may control many accounts, but only one public address may exist per account.
+
+Since account owners are the only ones who can modify their own account chains, contention happens on a per-account basis. If account A attempts a double spend that must be resolved by the network, account B can still make transactions as normal. Transactions are processed independently and asynchronously.
+
 ### Blocks
 
+In traditional blockchain-based cryptocurrencies like Bitcoin, a block is a group of transactions. In Nano, a block is a single transaction, so the term “block” and “transaction” are often used interchangeably. "Transaction" specifically refers to the action, while block refers to the digital encoding of the transaction. Transactions are signed by the private-key belonging to the account on which the transaction is performed. 
+
+An example of a Nano block:
+```
+{
+  "hash": "FF0144381CFF0B2C079A115E7ADA7E96F43FD219446E7524C48D1CC9900C4F17",
+  "difficulty": "ffffffe1278b3dc6", // since V21.0
```

Comment:
These two are obtained from the details below, perhaps leave only the insides of "block" below

Reply:
Good point, fixed